src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionExecutableStatementCountTest.java

```diff
@@ -122,10 +122,10 @@ public void testLambdas() throws Exception {
             "7:22: " + getCheckMessage(ExecutableStatementCountCheck.class, MSG_KEY, 2, 1),
         };
 
-        final List<String> expectedXpathQueries = Arrays.asList(
-            "/COMPILATION_UNIT/CLASS_DEF[./IDENT"
-            + "[@text='SuppressionXpathRegressionExecutableStatementCountLambdas']]"
-            + "/OBJBLOCK/VARIABLE_DEF[./IDENT[@text='c']]/ASSIGN/LAMBDA"
+        final List<String> expectedXpathQueries = List.of(
+                "/COMPILATION_UNIT/CLASS_DEF[./IDENT"
+                        + "[@text='SuppressionXpathRegressionExecutableStatementCountLambdas']]"
+                        + "/OBJBLOCK/VARIABLE_DEF[./IDENT[@text='c']]/ASSIGN/LAMBDA"
         );
```

Comment:
@nrmancuso ,
Really sorry for getting involved but could you please tell me what the error was with this file ?
This test has been created as part of my PR #14578 and knowing the error will help avoid it going further.

I was going to use a singletonList of Collections here if checkstyle threw any errors

Reply:
Read the build failure from your PR: https://app.circleci.com/pipelines/github/checkstyle/checkstyle/24261/workflows/30c63bd9-0623-4723-814f-1afb0c072212/jobs/534230/artifacts

Reply:
@MANISH-K-07 if you use Intellij, it will highlight these issues for you in the IDE.

Reply:
> Read the build failure from your PR: https://app.circleci.com/pipelines/github/checkstyle/checkstyle/24261/workflows/30c63bd9-0623-4723-814f-1afb0c072212/jobs/534230/artifacts

Right, even I missed that thinking it to be a common failure known to us

> I was going to use a singletonList of Collections here if checkstyle threw any errors

Should have gone with this, there would not have been an error....

> @MANISH-K-07 if you use Intellij, it will highlight these issues for you in the IDE.

I have tried all the three IDE (begun with Intellij IDEA, then experimented with Eclipse) 
and finally settled on Apache Netbeans 20 as of now